.eslintrc.json

```diff
@@ -8,6 +8,7 @@
         "eslint:recommended",
         "plugin:react/recommended"
     ],
+    "parser": "babel-eslint",
```

Comment:
Your build is failing because of this, does it need to be added to the project's dev dependencies?